macrel/main.py

```diff
@@ -332,14 +333,24 @@ def main(args=None):
         return
 
     with tempfile.TemporaryDirectory(dir=args.tmpdir) as tdir:
+        from .output import readme_output_abundance_mode,readme_output_contigs_mode,\
```

Comment:
I feel `README.md` is better (making sure that the file is indeed Markdown)

Reply:
Now it is done all in markdown.